cds_ils/migrator/api.py

```diff
@@ -360,12 +86,12 @@ def process_document_dump(data, source_type=None):
 
     :param data: Dictionary for representing a single record and files.
     """
-    recorddump = CDSRecordDump(
+    record_dump = CDSRecordDump(
         data,
         source_type=source_type,
     )
     try:
-        CDSDocumentDumpLoader.create(recorddump)
+        CDSDocumentDumpLoader.create(record_dump)
```

Comment:
Maybe if we pass the DumpLoader class as argument so that we can use this function with SeriesDumpLoader too?

Reply:
I thought about it, but I decided that it will be more readable when having a separate class for different record types